provider/unit_test_store.go

```diff
@@ -26,6 +28,10 @@ func (u UnitTestProvider) Config() pc.SerializedConfig {
 	return u.ProviderConfig
 }
 
+func (u UnitTestProvider) Check() (bool, error) {
```

Comment:
Should this be `CheckHealth()` now?

Reply:
That's odd. I thought I globally renamed this method. I'm not sure why it didn't catch these. Thank you!